images/python/README.md

```diff
@@ -38,6 +38,12 @@ docker pull cgr.dev/chainguard/python:latest
 docker pull cgr.dev/chainguard/python:latest-dev
 ```
 
+Note that in order to access the shell in the `python:latest-dev` image, you'll need to include an `--entrypoint` option, as in the following example.
+
+```sh
+docker run -it --entrypoint bin/bash chainguard/python:latest-dev
```

Comment:
I'd probably use absolute vs relative path in case workdir changes at any point:
```suggestion
docker run -it --entrypoint /bin/bash chainguard/python:latest-dev
```